### PR TITLE
Token underscoring

### PR DESCRIPTION
**Pull request by dsavchenko**

### CODE REVIEW

**Comment by dsavchenko** [comment]
@okolo just FYI, here is an implementation of token renaming. You will get "_token" request argument on the backend side.
It won't conflict with a  "token" parameter in nb if there is any

**Review by burnout87 — APPROVED** [review]

File: dispatcher_plugin_nb2workflow/queries.py [hunk 1]
```diff
@@ -32,10 +32,13 @@ def construct_parameter_lists(backend_param_dict, ontology_path):
...
                             "http://odahub.io/ontology#EndTime": "T2",
                             "http://odahub.io/ontology#AstrophysicalObject": "src_name",
-                            "ThisNameShouldNotExist": "token"
+                            "ThisNameShouldNotExist": "token",
                           }
-    par_name_substitution = {}
-
+    par_name_substitution = {'token': '_token'}
+    # There are two name substitutions for "*token": 
+    # 1) if backend notebook defines parameter named "token", it will be renamed by general mechanism (appending "_rename")
```
Comment (burnout87) [inline dispatcher_plugin_nb2workflow/queries.py:39]: where is this renaming taking place ?

**Review by burnout87 — COMMENTED** [review]
just one question

File: dispatcher_plugin_nb2workflow/queries.py (hunk 1 above)
Comment (dsavchenko) [reply dispatcher_plugin_nb2workflow/queries.py:39]: `par_name_substitution` dict gets populated lower depending on conditions (either rename or keep the name) for all parameters.

Then, it gets used at L116-117 to pass proper names to backend

**Review by volodymyrss — APPROVED** [review]

**Pull request merged**